_projects/eandb_project.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: page_project
+title: Energy Efficiency and Load Balancing
+date: 2016-02-17
+updated:
+navbar: Research
+subnavbar: Projects
+footer: true
+project_url:
+status: running
+topics:
+  - prog_lang
+keywords:
+  -
+head: keller_tesser_r
+members:
+  - padoin_e
+  - navaux_p
+  - mendes_c
+  - kale_s
+  - mehaut_j
+---
+
+## Research topic and goals
+
+The power consumption of High Performance Computing (HPC) systems is an increasing concern
+as large-scale systems grow in size and, consequently, consume more energy. 
+In response to this challenge, we propose new energy-aware load balancers that aim at reducing the 
+energy consumption of parallel platforms running imbalanced scientific applications without degrading 
+their performance. 
+Our research explores dynamic load balancing, low power manycore platforms and DVFS techniques in 
+order to reduce power consumption.
+
+## Results for 2014/2015
+
+#### Load Balancing  
+In this work we propose the improvement of the performance and scalability of parallel seismic
+wave models through dynamic load balancing. 
+These models suffer from load imbalance for two reasons. 
+First, they add a specific numerical condition at the borders of the domain, in order to
+absorb the outgoing energy. 
+The decomposition of the domain into a grid of subdomains, which are distributed among tasks, creates 
+load differences between the tasks that simulate the borders and those responsible for the central subdomains. 
+Second, the propagation of waves in the simulated area changes the workload on the subdomains on different time-steps. 
+Therefore causing dynamic load imbalance. 
+In order to evaluate the use of dynamic load balancing, we ported a seismic wave simulator to Adaptive MPI, 
+to benefit from its load balancing framework. Our experimental results
+show that dynamic load balancers can adapt to load variations during the application’s execution
+and improve performance by 36%.
+This work was presented in the PDP 2014 conference {% cite KellerTesserEtAl2014 --file jlesc.bib %}. 
+An extended version will be publishedin the International Journal of High Performance Computing and Applications.
```

Comment:
That's ok, you can also cite paper with "submitted" or "accepted" (like this one).